presentations/proposed/scipy2023/proposal.md

```diff
@@ -0,0 +1,86 @@
+# SciPy 2022 Proposal
+
+Jul 11-16, 2023
+
+https://www.scipy2023.scipy.org/present
+
+## Title
+
+Scientific Python---Year in Review
+
+## Session type
+
+Talk (30 minutes)
+
+## Abstract (100 words)
+
+The Scientific Python project aims to better coordinate the ecosystem and grow the community.
+This talk focuses on our efforts to better coordinate project development, and to improve shared infrastructure.
+Come hear about our recent progress and our plans for the coming year!
+
+## Description (500 words)
+
+The Scientific Python project's vision is to help pave the way toward a vibrant, unified scientific Python community.
+It focuses its efforts along two primary axes: _(i)_ to create a joint community around all scientific Python projects
+and _(ii)_ to support maintainers by building cross-cutting technical infrastructure and tools.
```

Comment:
isn't `all` a bit too broad? "Scientific python projects" hasn't really been defined, so it's not clear whether you mean open source, libraries, research code, or random scientific one-person projects.